source/styles/_base.scss

```diff
@@ -22,3 +22,28 @@ h3 {
   margin: 0;
   color: colors.$shark;
 }
+
+.column {
+  display: flex;
+  flex-direction: column;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+}
+
```

Comment:
I assume these definitions overriding Semantic-UI styles?
If so, consider moving them to _overrides.scss in the future